src/test/java/ru/training/at/hw3/main/differentelementspage/DifferentElementsPage.java

```diff
@@ -0,0 +1,60 @@
+package ru.training.at.hw3.main.differentelementspage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class DifferentElementsPage {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @FindBy(xpath = "//label[1]/input")
+    WebElement waterCheckBox;
+
+    @FindBy(xpath = "//label[3]/input")
+    WebElement windCheckBox;
+
+    @FindBy(xpath = "//label[4]/input[@name='metal']")
+    WebElement selenRadio;
+
+    @FindBy(xpath = "//div[@class='colors']")
+    WebElement colorsMenu;
+
+    @FindBy(xpath = "//div[@class='colors']/select/option[4]")
+    WebElement yellowDropdown;
+
+    public DifferentElementsPage(WebDriver driver, WebDriverWait wait) {
+        this.driver = driver;
+        this.wait = wait;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void selectElements() {
```

Comment:
It's very specific method with main purpose to do action from homework. Let's write something flexible. Also if we are trying to write that methods as interaction steps they have to be atomic and do just one action - click, sendKeys, getText and so on.